src/sumo.erl

```diff
@@ -16,6 +16,12 @@
 %%% @end
 %%% @copyright Inaka <hello@inaka.net>
 %%%
+
+
+%%% code added to check elvis
+ct:pal("trying to test elvis-server in production trow this repo example and this long line ~p ~p",[V1,V2]),
```

Comment:
Line 22 is too long: ct:pal("trying to test elvis-server in production trow this repo example and this long line ~p ~p",[V1,V2]),.